dts-generator/src/main/java/com/telerik/dts/DtsApi.java

```diff
@@ -757,7 +757,7 @@ private Type getReturnType(Method m) {
             Matcher matcher = methodSignature.matcher(signature.getSignature());
             if(matcher.matches()) {
                 String returnSignature = matcher.group(2);
-                if(returnSignature.equals("V")){
+                if(returnSignature.equals("V") || returnSignature.contains("^")){
```

Comment:
It would be better if you use the following regex to match "V" or "V^{anything};", as it may happen to have the same caret as part of another signature but not void. So I would propose the following:

add the following regex right above the constructor: 
```Java
private Pattern isVoid = Pattern.compile("V(\\^.*\\;)?");
```
and then make the if check like this:
```Java
if(isVoid.matcher(returnSignature).matches())
```

Reply:
@vtrifonov 
`Added requested Pattern isVoid regex check` https://github.com/NativeScript/android-dts-generator/pull/33/commits/5b9a74f4717124d44efb01d808ab35f9751c9d5e